src/main/resources/static/service-info.json

```diff
@@ -0,0 +1,77 @@
+{
+  "id": "uk.ebi.eva.eva-seqcol",
+  "name": "Implementation of the GA4GH Sequence collection retrieval API",
+  "type": {
+    "group": "org.ga4gh",
+    "artifact": "seqcol",
+    "version": "0.0.0"
+  },
+  "organization": {
+    "name": "European Variation Archive",
+    "url": "https://www.ebi.ac.uk/eva"
+  },
+  "contactUrl": "mailto:eva-helpdesk@ebi.ac.uk",
+  "documentationUrl": "https://www.ebi.ac.uk/eva/....",
+  "updatedAt": "08-24-2023",
+  "environment": "dev",
+  "version": "tagged version",
```

Comment:
As a future improvement maybe `version` and `updatedAt` should be filled dynamically, even if the rest isn't.

Also @tcezard, what's the difference between this `version` and `type.version`?  This one is the code version and the other is the spec version?

Reply:
For the versions: 
 - [version](https://github.com/ga4gh-discovery/ga4gh-service-info/blob/develop/service-info.yaml#L101) is the version of the service
 - [type.version](https://github.com/ga4gh-discovery/ga4gh-service-info/blob/c8481aa5f5aa499cd2a6af41a3f29564000bf672/service-info.yaml#L119) is the version of the API